Automattic/wp-calypso
# ActivityLog: Filter bar visual tweaks and fixed tablet layout

Solves https://github.com/Automattic/jetpack-backup-team/issues/259 and https://github.com/Automattic/jetpack-backup-team/issues/260

## Proposed Changes

* Fix layout weirdness in tablets
* Fix Activity Type vertical overflow
* Remove ellipsis in the clear buttons in some resolutions
* Clear buttons now have a border-left to avoid having two icons in close proximity

![CleanShot 2023-11-29 at 21 17 11](https://github.com/Automattic/wp-calypso/assets/1488641/39e66b6f-a792-40d8-987d-b3f810fe7236)


## Testing Instructions



* Spin up a Calypso and Jetpack Cloud live branch
* Navigate to the Activity Log.
* Play around with the filters and ensure it is not breaking UI when date range and activity types.
* To test the changes, I recommend you to play around with different resolutions (using Chrome Developer Tools).

## Pre-merge Checklist
